newsfragments/25507.rst

```diff
@@ -0,0 +1,3 @@
+Removal of experimental Smart Sensors
+
+Smart Sensors  were added in 2.0 and deprecated in favour of Deferable operators in 2.2 and have now been removed.
```

Comment:
```suggestion
Smart Sensors were added in 2.0 and deprecated in favour of Deferable operators in 2.2 and have now been removed.
```

(nit)

This should also be `25507.significant.rst`.

Reply:
My other question about newsfragments if we should have any formatting in them (lists, headings etc?)

Reply:
"should", no. "can", for significant yep!

Reply:
A little longer answer: Non-significant can only have a single line. Significant can be a single line, or multi-line rst.

Reply:
PR to improve the precommit: #25536